extra/grpc/autogptq/autogptq.yml

```diff
@@ -0,0 +1,86 @@
+name: autogptq
+channels:
+  - defaults
+dependencies:
+  - _libgcc_mutex=0.1=main
+  - _openmp_mutex=5.1=1_gnu
+  - bzip2=1.0.8=h7b6447c_0
+  - ca-certificates=2023.08.22=h06a4308_0
+  - ld_impl_linux-64=2.38=h1181459_1
+  - libffi=3.4.4=h6a678d5_0
+  - libgcc-ng=11.2.0=h1234567_1
+  - libgomp=11.2.0=h1234567_1
+  - libstdcxx-ng=11.2.0=h1234567_1
+  - libuuid=1.41.5=h5eee18b_0
+  - ncurses=6.4=h6a678d5_0
+  - openssl=3.0.11=h7f8727e_2
+  - pip=23.2.1=py311h06a4308_0
+  - python=3.11.5=h955ad1f_0
+  - readline=8.2=h5eee18b_0
+  - setuptools=68.0.0=py311h06a4308_0
+  - sqlite=3.41.2=h5eee18b_0
+  - tk=8.6.12=h1ccaba5_0
+  - wheel=0.41.2=py311h06a4308_0
+  - xz=5.4.2=h5eee18b_0
+  - zlib=1.2.13=h5eee18b_0
+  - pip:
```

Comment:
Why are we ver locking these packages? (Just asking to learn mostly)

Reply:
Hi, there. We want to make sure the environment stable. You may know code will not working well after the dependencies were updated. If we do not give the specific version of the dependencies. It will download the latest version of the packages. So,  it may cause conflicts. For example, you can see commit id of packages in `go.mod`. It does same thing.

Reply:
(Also a added sorry, the way I added that comment seems to make github thing I need to review it, deeply sorry that was not a review question, just a question to learn)

Reply:
Never mind.